### ISSUE TITLE
InvalidCacheBackendError: Could not find backend 'django_redis.cache.RedisCache': No module named 'django_redis.cache'

### ISSUE THREAD

**Issue by REXIANN** [jazzband/django-redis#489]
Hi, 

I tried to use django-redis and I got this error when I entered the page where the redis cache has set. 

Request Method: | GET
-- | --
http://127.0.0.1:8000/
2.2
InvalidCacheBackendError
Could not find backend 'django_redis.cache.RedisCache': No module named 'django_redis.cache'
/Users/jinhyuck/Documents/django_redis/venv/lib/python3.6/site-packages/django/core/cache/__init__.py in _create_cache, line 53
/Users/jinhyuck/Documents/django_redis/venv/bin/python3
3.6.8
['/Users/jinhyuck/Documents/django_redis',  '/Library/Frameworks/Python.framework/Versions/3.6/lib/python36.zip',  '/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6',  '/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/lib-dynload',  '/Users/jinhyuck/Documents/django_redis/venv/lib/python3.6/site-packages']
Wed, 6 Jan 2021 22:13:29 +0900

--------

I have installed django-redis with pip and all related libraries meet django-redis' dependencies. 

(venv) ➜  django_redis git:(master) ✗ pip list
Package      Version
------------ -------
asgiref      3.3.1
Django       2.2
django-redis 4.12.1
pip          20.3.3
pytz         2020.5
redis        3.5.3
redis-server 6.0.9
setuptools   40.6.2
sqlparse     0.4.1

I also added Redis cache setting into django's settings.py file.

# Redis cache setting

CACHES = {
    "default": {
        "BACKEND": "django_redis.cache.RedisCache",
        "LOCATION": "redis://127.0.0.1:6379/1",
        "OPTIONS": {
            "CLIENT_CLASS": "django_redis.client.DefaultClient",
        }
    }
}

Surely, 'django_redis' has added  in INSTALLED_APPS list.

INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    
    # install apps
    'articles',
    'accounts',
    
    # django_redis 
    'django_redis',
]

and I found out that there is a cache.py file in vertualenv folder. When I type 'ls' command on path 'venv/lib/python3.6/site-packages/django_redis' ...

django_redis git:(master) ✗ ls
__init__.py   cache.py      compressors   hash_ring.py  serializers
__pycache__   client        exceptions.py pool.py       util.py

I have no idea why django returns an error 'No module name d'djano_redis.cache'. You can see venv folder path has added to python path. Anyone could help me please...?

**Comment by WisdomPill** [comment]
Are you sure that you are starting django from the virtual environment?

What is the name of your project? Maybe you have the same name as the library so it will find you project before the library itself.

There has not been any releases since may, many users and I use `dingo_redis` on daily basis.

**Comment by REXIANN** [comment]
Thanks for your advice. 

Yes, you were right. My project name was also `django_redis` so I changed it. 

But it still doesn't work with same error message...

Found out that the error occurs at line 50, `import_string(backend)` statement in /Users/jinhyuck/Documents/django_redis/venv/lib/python3.6/site-packages/django/core/cache/init.py file. 

 I've also installed `django-redis` library on global python path and it didn't work either. 

Well, It could be a problem of mine if many users use this library including you and there hasn't been any new releases..

I'll figure it out. thanks again @WisdomPill !
